ipydatagrid/_version.py

```diff
@@ -1,4 +1,4 @@
 # Copyright (c) Bloomberg.
 # Distributed under the terms of the Modified BSD License.
 
-__version__ = "1.1.16"
```

Comment:
We should bump this separately

Reply:
Done

Reply:
The package.json still has the new version I think